marimbabot_planning/src/move_to_pose.cpp

```diff
@@ -0,0 +1,49 @@
+#include <iostream>
+#include <ros/ros.h>
+#include <moveit/move_group_interface/move_group_interface.h>
+#include <tf2_ros/transform_listener.h>
+#include <geometry_msgs/TransformStamped.h>
+
+
+int main(int argc, char **argv)
+{
+    ros::init(argc, argv, "marimba_move");
+    ros::NodeHandle node_handle;
+    ros::AsyncSpinner spinner(1);
+    spinner.start();
+
+    static const std::string PLANNING_GROUP = "arm_with_tcp";
+    moveit::planning_interface::MoveGroupInterface move_group_interface(PLANNING_GROUP);
+
+    const moveit::core::JointModelGroup* joint_model_group =
+        move_group_interface.getCurrentState()->getJointModelGroup(PLANNING_GROUP);
+
+
+    tf2_ros::Buffer tfBuffer;
+    tf2_ros::TransformListener tfListener(tfBuffer);
+    ros::Duration(1).sleep();
+
+    geometry_msgs::TransformStamped transformStamped;
+    try{
+        transformStamped = tfBuffer.lookupTransform("world", "diana_gripper/tcp", ros::Time(0));
```

Comment:
Are the frames correct? Is moveit running in the `world` frame?

Reply:
Yes, so far it worked, we got the motion.